src/components/ui/dropzone.tsx

```
@@ -134,9 +134,9 @@ const getRootError = (
     const errors = errorCodes.map((error) => {
         switch (error) {
             case "file-invalid-type":
-                const acceptedTypes = Object.values(limits.accept ?? {})
-                    .flat()
-                    .join(", ");
+                const acceptedTypes = Array.from(
+                    new Set(Object.values(limits.accept ?? {}).flat()),
+                ).join(", ");
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)